src/core/core.controller.js

```diff
@@ -322,10 +323,11 @@ module.exports = function(Chart) {
 			}
 
 			// Clip out the chart area so that anything outside does not draw. This is necessary for zoom and pan to function
-			this.chart.ctx.save();
-			this.chart.ctx.beginPath();
-			this.chart.ctx.rect(this.chartArea.left, this.chartArea.top, this.chartArea.right - this.chartArea.left, this.chartArea.bottom - this.chartArea.top);
-			this.chart.ctx.clip();
+			var context = this.chart.ctx;
```

Comment:
I guess the *easier to read* is a personal thing :) (to me it's easier to read many `ctx` than `context`). That's also because I like short names and `ctx` is commonly used. Not sure about the renaming argument, it would mean that we have to always pick local variable names different from class members (e.g `var gridLines = options.gridLines;`, `var scaleLabel = options.scaleLabel;`, etc...)?

Nevermind, if @etimberg is fine with that, I'm as well :)